docs/process/okrs.md

```diff
@@ -23,3 +23,10 @@ When establishing OKRs, we adopt a scientific approach to avoid setting unattain
 We closely monitor OKR progress through weekly updates in our dedicated Slack channel and bi-weekly check-ins to provide teams and individuals with valuable feedback and refine our approach. We use a color code to indicate progress and capacity during these check-ins: green signifies being on track (0.7 to 1.0), yellow represents areas that need attention (0.4 to 0.6), and red indicates being at risk (0 to 0.3). OKRs are not static mandates. We evaluate our OKRs to ensure they align with the most important work at Flexpa and adjust them if necessary. OKRs are scored, and new ones are agreed upon at our quarterly in-person retreat [link], fostering reflection, alignment, and goal setting. 
 
 We have embraced OKRs to drive our mission of refactoring healthcare. We adopted OKRs early on, recognizing the importance of everyone understanding our top priorities and how to measure our impact. We set OKRs at the company and team levels, consistently reviewing our progress and learning from challenges. OKRs serve as a catalyst for meaningful conversations, valuable feedback, and recognition within our teams, enhancing both performance and our company culture. We firmly believe that OKRs, along with regular progress tracking and check-ins, are key factors in our success, and we constantly strive to leverage them even better.
+
+### Each quarter, there is one owner of each Level 2 Objective and accompanying KRs. They are responsible for moving the needle on their Objective and accompanying KRs, and they report on a weekly basis to the Level 1 Objective owner through 2 mechanisms.
```

Comment:
```suggestion
### Each quarter, there is one owner of each Level 2 Objective and accompanying KRs. They are responsible for moving the needle on their Objective and accompanying KRs, and they report on a weekly basis to the Level 1 Objective owner through 2 mechanisms.
```

This should be a level heading like `## Reporting` and then this should be first paragraph.

I think also that this document should outline what "Level 2 Objectives" are - the idea is first introduced here, but isn't explained.

Reply:
Good comments. Good progress.